idaes/gas_solid_contactors/unit_models/tests/test_BFB.py

```diff
@@ -507,23 +543,23 @@ def test_solve(self, iron_oc):
     def test_solution(self, iron_oc):
         assert (pytest.approx(0.140, abs=1e-2) ==
                 iron_oc.fs.unit.velocity_superficial_gas[0, 0].value)
-        assert (pytest.approx(1.349, abs=1e-2) ==
+        assert (pytest.approx(1.028, abs=1e-2) ==
```

Comment:
Are these changes expected?

Reply:
Also, these tolerances are very loose - we should be checking to close to solver tolerance in most cases (~`rel=1e-5`).

Reply:
I believe the new values are okay. In the last couple commits, I noticed that some scaling factors changed in magnitude. This may be due to the unit fixes that occurred earlier, although I'm not sure why the tests didn't fail then. The only change I made to the model equations was to correct a small error in how `reform_var_5` is smoothed on line 1228 in the most current file version.

Reply:
I've confirmed that the new values of the variables are correct. I cross checked against an older version of the model and they match. The changes are as a result of updating the orifice pressure drop from 3E5 Pa to 3E3 Pa.